recipes/sphinx_removed_in/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "sphinx_removed_in" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0588239cb534cd97b1d3900d0444311c119e45296a9f73f1ea81ea81a2cd3db1
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - sphinx
+
+test:
+  imports:
+    - sphinx_removed_in
+    - tests
```

Comment:
These test are actually run without the source in the directory.

Therefore, you cannot use the tests directory.

Typically on conda-forge, for pure python packages, we simply test with an "import". Therefore all lines below `sphinx_removed_in` are not necessary for the tests section